MovieApp/MovieApp.xcodeproj/project.pbxproj

```diff
@@ -78,6 +107,17 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		CE530D941D995FC5003A708F /* Help */ = {
+			isa = PBXGroup;
+			children = (
+				CE530D951D996043003A708F /* FeedDownloader.h */,
+				CE530D961D996043003A708F /* FeedDownloader.m */,
+				CE530D981D9960A2003A708F /* BoxOfficeFeedDownloader.h */,
+				CE530D991D9960A2003A708F /* BoxOfficeFeedDownloader.m */,
+			);
+			name = Help;
+			sourceTree = "<group>";
+		};
```

Comment:
Help group should be renamed to Helpers or Utilities to keep some naming standard accross the app.